File: app/src/main/java/com/jqorz/test/popup/TestPopup.java
package com.jqorz.test.popup;

import android.content.Context;
import android.view.View;

import com.jqorz.test.R;

import razerdp.basepopup.BasePopupWindow;

/**
 * copyright datedu
 *
 * @author jqorz
 * @since 2019/11/1
 */
public class TestPopup extends BasePopupWindow {

    public TestPopup(Context context) {
        super(context);
    }

    @Override
    public View onCreateContentView() {
        return createPopupById(R.layout.layout_popup);
    }
}

File: app/src/main/java/com/jqorz/test/windows/popup/TestPopup.kt
package com.jqorz.test.windows.popup

import android.content.Context
import android.view.Gravity
import android.view.View
import com.jqorz.test.R
import razerdp.basepopup.BasePopupWindow

/**
 * copyright datedu
 *
 * @author jqorz
 * @since 2019/11/1
 */
class TestPopup(context: Context?) : BasePopupWindow(context) {
    override fun onCreateContentView(): View {
        return createPopupById(R.layout.layout_popup)
    }

    init {
        setPopupGravity(GravityMode.RELATIVE_TO_ANCHOR, Gravity.END or Gravity.CENTER_VERTICAL)
    }
}